gate-web/gate-web.gradle

```diff
@@ -43,6 +43,7 @@ dependencies {
   compileOnly spinnaker.dependency("lombok")
 
   testCompile project(":gate-ldap") // TODO: Move system tests to own module
+  testCompile project(":gate-basic")
```

Comment:
I think you'll want this to be `compile project(":gate-basic")`, otherwise the functionality won't be included in the compiled jar.

Reply:
For that case, all gate auth modules are already included as runtime dependencies, and the list of those is built from `gradle.properties`: https://github.com/spinnaker/gate/pull/675/files/36ed77203133a36a5a25e83b77e67965138db711#diff-503f218d646c10f484fdc9d6315bf2e3R2
These test dependencies are only there because of some tests written against ldap and basic implementations.

Reply:
Oh, right on thanks!